config/config.go

```diff
@@ -484,10 +484,16 @@ type HttpServerOptionsConfig struct {
 	// Enabled WebSockets and server side events support
 	EnableWebSockets bool `json:"enable_websockets"`
 
-	// Deprecated. SSL certificates used by Gateway server.
+  
+	// Deprecated: Use `ssl_certificates`instead.
+	// SSL certificates used by Gateway server.
```

Comment:
As this is deprecated, should we just remove this line? If it doesn't do anything we don't need to say what it no longer does.

```suggestion
```